Makefile

```diff
@@ -96,22 +105,20 @@ codegen-engine-tonic:
 		--tonic_out=$(ODF_CRATE_DIR)/src/engine/grpc_generated \
 		--tonic_opt=compile_well_known_types
 
-	$(RUSTFMT) $(ODF_CRATE_DIR)/src/engine/grpc_generated/engine.rs
-	$(RUSTFMT) $(ODF_CRATE_DIR)/src/engine/grpc_generated/engine.tonic.rs
-
 	$(call add_license_header, "$(ODF_CRATE_DIR)/src/engine/grpc_generated/engine.rs")
 	$(call add_license_header, "$(ODF_CRATE_DIR)/src/engine/grpc_generated/engine.tonic.rs")
 
 
 .PHONY: codegen-graphql
 codegen-graphql:
 	python $(ODF_SPEC_DIR)/tools/jsonschema_to_rust_gql.py $(ODF_SPEC_DIR)/schemas \
-		| $(RUSTFMT) > src/adapter/graphql/src/scalars/odf_generated.rs
+		> src/adapter/graphql/src/scalars/odf_generated.rs
 	$(call add_license_header, "src/adapter/graphql/src/scalars/odf_generated.rs")
 
 
 .PHONY: codegen
 codegen: codegen-odf-serde codegen-engine-tonic codegen-graphql
+	cargo fmt --all
```

Comment:
what was the reason for not piping through `rustfmt`?

this change will mean that if I run `make codegen-graphql` I will have a poorly formatted file

(although it's very rare that I run individual targets and not the top-level `codegen`)

Reply:
Honesty, just to cover all files since these lines didn't perform actual formatting
```shell
	$(RUSTFMT) $(ODF_CRATE_DIR)/src/engine/grpc_generated/engine.rs
	$(RUSTFMT) $(ODF_CRATE_DIR)/src/engine/grpc_generated/engine.tonic.rs
```

> this change will mean that if I run make codegen-graphql I will have a poorly formatted file

makes sense, I'll fix that

Reply:
returned piping, don